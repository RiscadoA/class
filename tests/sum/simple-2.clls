proc main() {
    cut {
        println("case x");
        case x of {
            |#C: println("close x 1"); close x
            |#W: println("wait x"); wait x; ()
        }
        |x : choice of { |#C: wait |#W: close }|
        println("#W x");
        #W x;
        println("close x 2");
        close x
    }
};;
