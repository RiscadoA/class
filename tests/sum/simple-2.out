#W x
close x 2
case x
wait x
